apps/wavelet/CMakeLists.txt

```diff
@@ -5,10 +5,23 @@
 # Generators in a simple app.
```

Comment:
The other ones get to rely on stuff that is intrinsically set up by the root CMakeLists.txt; this one is intentionally standalone. But yeah, the goal is to greatly reduce the extra complication here.